src/main/java/reposense/util/FileUtil.java

```diff
@@ -85,7 +86,7 @@ public static void zipFoldersAndFiles(List<Path> pathsToZip,
     /**
      * Writes the JSON file representing the {@code object} at the given {@code path}.
      */
-    public static void writeJsonFile(Object object, String path) {
+    public static Optional<Path> writeJsonFile(Object object, String path) {
```

Comment:
Same here. At the same time, do include what's being returned if there's issues writing into file

Reply:
Done. I've updated the Javadoc to indicate the return value in both the normal and exceptional cases.